### PR TITLE
OTC-928: fixed discrepancy in search parameters for Families and Insurees

### PR DESCRIPTION
**Pull request by lruzicki**
Ticket:
https://openimis.atlassian.net/browse/OTC-928

Changes:
- change translations
- fixed displayed value in insuree summary

Note:
Lokalise is changed too

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -41,15 +41,15 @@
...
+  "insuree.InsureeGender.null": "Any",
   "insuree.InsureeGender.M": "Male",
   "insuree.InsureeGender.F": "Female",
   "insuree.InsureeGender.O": "Other",
   "insuree.InsureeOfficerPicker.label": "Officer",
   "insuree.InsureeOfficer.null": "Any",
   "insuree.InsureeOfficer.none": "None",
   "insuree.InsureeMaritalStatus": "Marital Status",
-  "insuree.InsureeMaritalStatus.null": "Unknown",
+  "insuree.InsureeMaritalStatus.null": "Any",
```
Comment (delcroip) [inline src/translations/en.json:52]:
```suggestion
  "insuree.InsureeMaritalStatus.null": "Not Specified",
```

**Review by delcroip — CHANGES_REQUESTED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (delcroip) [reply src/translations/en.json:52]: unless there were a reason, please commit sugestion

Comment (lruzicki) [reply src/translations/en.json:52]: This one is for picking all options in search form. For not specified is:
`"insuree.InsureeMaritalStatus.N": "Not Specified",`

**Review by dragos-dobre — APPROVED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (delcroip) [reply src/translations/en.json:52]: I see, but it is more empty than any....

**Review by delcroip — APPROVED** [review]

**Pull request merged**